### DISCUSSION TITLE
游戏打不开咋回事？

### DISCUSSION THREAD

**Discussion by TXSQ2009** [Meloong-Git/PCL#2270] [提问] [answered]
[错误报告-2023-07-17_下午_1.09.34.zip](https://github.com/Hex-Dragon/PCL2/files/12066921/-2023-07-17_._1.09.34.zip)

**Comment by iPodtouch6557** [comment]
没传上来

**Reply by TXSQ2009** [reply]
好了，没想到没上传成功

**Comment by SALTWOOD** [comment]
错误报告没传好，重新传一次

**Reply by TXSQ2009** [reply]
不好意思，现在上传好了

**Comment by zkitefly** [comment, marked as answer]
当前你所安装的 Forge 47.1.14 过高，OptiFine_I6_pre1 最高只兼容至 Forge 47.0.35

https://zkitefly.github.io/optifine-forge-support-list/game/1.20.1#:~:text=HD_U_I6_pre1

解决方法：

- 降级 Forge 
- 卸载 Optifine

**Reply by TXSQ2009** [reply]
谢谢

**Comment by SALTWOOD** [comment]
此版本Forge和Optifine不兼容
降级Forge或卸载opti
